sdk/tests/conformance/textures/misc/tex-sub-image-2d-bad-args.html

```diff
@@ -107,6 +107,15 @@
     wtu.glErrorShouldBe(gl, gl.INVALID_OPERATION, "type not same as original RGBA 4_4_4_4");
 }
 
+var largeCanvas = document.createElement("canvas");
```

Comment:
The test itself looks fine, but it doesn't fit in this tex-sub-image-2d-bad-args.html as all args are good here.